redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/actions/CertRotateInRedBeamsAction.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.actions;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsContext;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedBeamsRequest;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerSuccess;
+
+@Component("CERT_ROTATE_STATE")
+public class CertRotateInRedBeamsAction extends AbstractRedbeamsStartAction<StartDatabaseServerSuccess> {
```

Comment:
Nit: `Beams` -> `beams`, so that it remains consistent with `AbstractRedbeamsStartAction`.